S3/English/ASS_31.ass

```diff
@@ -85,7 +85,7 @@ Dialogue: 0,0:03:24.40,0:03:27.35,Default,,0,0,0,,Started his first dance in the
 Dialogue: 0,0:03:28.15,0:03:30.10,Default,,0,0,0,,He was supposed to
 Dialogue: 0,0:03:32.56,0:03:33.10,Default,,0,0,0,,Damao
 Dialogue: 0,0:03:33.14,0:03:34.10,Default,,0,0,0,,Could you help me with...
-Dialogue: 0,0:03:35.65,0:03:38.65,Default,,0,0,0,,{\c&HAAA4D7&\pos(1215,275)}PPT design: 60￥/hour\NResume layout: 1000￥/hour\NLogo design: 300￥/hour\NPoster design: 400￥/hour\NOthers 200￥/hour
+Dialogue: 0,0:03:35.65,0:03:38.65,Default,,0,0,0,,{\c&HAAA4D7&\pos(1215,275)}Presentation design: 60￥/hour\NResume layout: 1000￥/hour\NLogo design: 300￥/hour\NPoster design: 400￥/hour\NOthers 200￥/hour
```

Comment:
Does this break the text alignment?

Reply:
I have no idea now. Perhaps I'll check it later if I have time. ~~(I'm pretty busy with my schoolwork.)~~